WordPress/src/main/java/org/wordpress/android/ui/posts/EditorBloggingPromptsViewModel.kt

```diff
@@ -36,7 +39,12 @@ class EditorBloggingPromptsViewModel
     private fun loadPrompt(site: SiteModel, promptId: Int) = launch {
         val prompt = bloggingPromptsStore.getPromptById(site, promptId).first().model
         prompt?.let {
-            _onBloggingPromptLoaded.postValue(Event(EditorLoadedPrompt(promptId, it.content, BLOGGING_PROMPT_TAG)))
+            val content = if (bloggingPromptsEnhancementsFeatureConfig.isEnabled()) {
+                bloggingPromptsEditorBlockMapper.map(it)
+            } else {
+                it.content
```

Comment:
Question: just for clarification, we are moving away from using the `content` directly from the response to have more flexibility and match what the web currently does, is that correct?